src/lib/components/NewLayeredMap/components/leafletUtils.js

```diff
@@ -0,0 +1,118 @@
+export const getShapeType = layer => {
```

Comment:
Hmmm, does this file really belong in the _components_-directory though? 🤔 I really don't feel like it is a component, and finding it in the root of the components-directory would be unexpecting, will it not?

I suggest that you rather make a _utils_ directory inside _NewLayeredMap_-directory, and put it there instead 😛 Thoughts? 😅